src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,15 +1,43 @@
-# TO-DO: complete the helpe function below to merge 2 sorted arrays
+# TO-DO: complete the helper function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
     # TO-DO
-    
-    return merged_arr
```

Comment:
popping off a zero index is not good time complexity.  much better to use pointers.